src/test/scala/cromwell/webservice/CromwellApiServiceIntegrationSpec.scala

```diff
@@ -11,8 +12,8 @@ import spray.json._
 import spray.testkit.ScalatestRouteTest
 
 class CromwellApiServiceIntegrationSpec extends FlatSpec with CromwellApiService with ScalatestRouteTest with Matchers {
-  def actorRefFactory = system
-  val workflowManager = TestActorRef(new WorkflowManagerActor(new LocalBackend))
+  def actorRefFactory = new TestWorkflowManagerSystem().actorSystem
+  val workflowManager = TestActorRef(new WorkflowManagerActor(new LocalBackend(system)))
```

Comment:
Honestly I just made the minimal changes I could to get them to work w/ my changes. Note that actorreffactory was already there, I just changed what it was being assigned. I have no idea why it was the way it was.